docs/bis/guide/fundamentals/attribution-guidelines.md

```diff
@@ -0,0 +1,71 @@
+# Attribution: Guidelines
+
+BIS offers multiple ways to capture attributes of Entities being modeled. It is the job of the schema designer to choose the most appropiate strategy to follow in every case. This article aims to provide general guidelines for the choosing among those strategies.
+
+## Introduction
+
+Not every attribute of an Entity needs to be persisted into a BIS repository. Some attributes can be derived from persisted data, therefore saving them in a BIS repository turns derived attributes into effectively cached data that needs to be kept in-sync as the more fundamental attributes change. This situation effectively applies to iTwin Native apps that directly store their data in BIS repositories.
+
+## Persisted Attributes
+
+The following sections focus on attributes that are meant to be persisted in a BIS repository.
+
+### First-class Attributes
```

Comment:
We mention perspectives but I worry that it will be missed or overlooked by someone not familiar with modeling perspectives.  I'm reading this from the perspective of someone who's trying to figure out where their properties should go.  From that persona's point of view mentioning that the property might be better suited for another perspective and linking to more docs about perspectives would round out the options of where to look when adding a new property